linux/sound.cpp

```diff
@@ -237,11 +237,8 @@ int CSound::process ( jack_nframes_t nframes, void* arg )
         {
             for ( i = 0; i < pSound->iJACKBufferSizeMono; i++ )
             {
-                pSound->vecsTmpAudioSndCrdStereo[2 * i] =
-                    (short) ( in_left[i] * _MAXSHORT );
-
-                pSound->vecsTmpAudioSndCrdStereo[2 * i + 1] =
-                    (short) ( in_right[i] * _MAXSHORT );
+	        pSound->vecsTmpAudioSndCrdStereo[2 * i] = in_left[i];
```

Comment:
Please check the code style. Here, the spaces seem not to match.

Reply:
Fixed.